pkg/cli/demo.go

```diff
@@ -189,6 +235,29 @@ func setupTransientServers(
 	}
 	urlStr := url.String()
 
+	// Start up the update check loop.
+	// We don't do this in (*server.Server).Start() because we don't want it
+	// in tests.
+	if shouldEnableTelemetry() {
+		s.PeriodicallyCheckForUpdates(ctx)
+		// If we allow telemetry, then also try and get an enterprise license for the demo.
+		if cliCtx.isInteractive {
+			db, err := gosql.Open("postgres", urlStr)
+			if err != nil {
+				return ``, ``, cleanup, err
+			}
+			// Perform license acquisition asynchronously to avoid delay in cli startup.
+			go func() {
+				defer db.Close()
+				err := getAndApplyLicense(db)
+				if err != nil {
+					msg := `error attempting to acquire demo license %+v. Enterprise feature are not enabled in this session`
```

Comment:
```go
fmt.Fprintf(stderr,
             "warning: unable to acquire a demo license: %+v\n"+
             "\nEnterprise features are not enabled in this session.")
```

1. use `fmt.Fprintf`  because Jordan is otherwise disabling logging
2. use `stderr` otherwise it doesn't work in tests
3. the error message may span multiple lines, so no period after its message
4. the additional hint/message should be separated with an extra newline otherwise it may merge with the multi-line error message
5. period at the end of the sentence